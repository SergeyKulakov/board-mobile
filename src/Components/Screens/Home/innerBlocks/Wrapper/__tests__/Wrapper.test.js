import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import { Wrapper as Component } from '../index'

describe('Home.Wrapper', () => {
  const defaultProps = {
    title: 'title',
    onClickShowAll: () => null,
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
