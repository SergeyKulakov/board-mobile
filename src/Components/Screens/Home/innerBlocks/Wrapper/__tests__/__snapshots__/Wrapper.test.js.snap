// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.Wrapper render snapshot 1`] = `
<ShadowBox
  style={
    Object {
      "marginBottom": 0,
      "paddingLeft": 0,
      "paddingRight": 0,
    }
  }
>
  <TouchableWithoutFeedback>
    <Styled(View)>
      <Styled(View)>
        <Component>
          title
        </Component>
        <TouchableOpacity
          activeOpacity={0.2}
          onPress={[Function]}
        >
          <Styled(Text)>
            View All
          </Styled(Text)>
        </TouchableOpacity>
      </Styled(View)>
    </Styled(View)>
  </TouchableWithoutFeedback>
</ShadowBox>
`;
